native/src/main/java/io/ballerina/stdlib/oracledb/utils/Utils.java

```diff
@@ -172,6 +173,12 @@ public static DataError throwInvalidParameterError(Object value, String sqlType)
                 valueName, sqlType));
     }
 
+    public static void throwUnsupportedArrayTypeError(BArray value) throws DataError {
+        throw new DataError(String.format("OracleDB module can only handle varray data types where both type name " +
```

Comment:
Use  UnsupportedTypeError

Reply:
I will update the sql dependency version to the latest and fix those error messages types in a separate PR as this PR addresses only the mix type support in the varray.